extension/docs/ManualCapture.md

```diff
@@ -52,54 +37,56 @@ Additionally, a [Charge](https://docs.commercetools.com/http-api-projects-paymen
 }
```

Comment:
interaction id should be 883592922122044A, right ?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/341/commits/a71be09ccf9cae86dd9a737db2b2314ac4b76377